### ISSUE TITLE
Unable to edit the html block after reloading the content back into the editor again

### ISSUE THREAD

**Issue by pichu0528** [ckeditor/ckeditor5#8743]
## 📝 Provide detailed reproduction steps (if any)

1. Insert the html snippet code block
2. Type some code, ie. alert('test'), and save the content
3. Reload the editor with the content that was just saved

### ✔️ Expected result

The html snippet block should be there with the code that was saved

### ❌ Actual result

There is no code block, and the code is wrapped in p tags.

## 📃 Other details

* Browser: Chrome
* OS: Windows
* First affected CKEditor version: 23.1.0
* Installed CKEditor plugins: htmlEmbed

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by Reinmar** [comment]
All works fine in the demo:

![](https://user-images.githubusercontent.com/156149/103512892-d8288980-4e69-11eb-9506-633ac3cb0ed1.png)

So you should check whether you're loading into the editor exactly the same content that you're getting from it.

**Comment by pichu0528** [comment]
Hello @Reinmar ,

Thank you for your response.
I think I have found the issue of why my editor was not behaving as it should because I have added the plugin of preserving all the div attributes from the [guide](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/deep-dive/conversion/conversion-preserving-custom-content.html#loading-content-with-all-attributes). I have adjusted the plugin where the editor is doing the upcast conversion. I have filtered out the div with class name of 'raw-html-embed' before doing the conversion. Now, it is working as expected!

Thank you!
